tests/syncreload/connections.csv

```diff
@@ -0,0 +1,18 @@
+action,ms,bytes,lang,code,repeat,minver,comment
+beforeany,0,0,q,system "sleep 5",1,,"wait for all procs to go up"
+before,0,0,q,hdbHandle:hopen`::41804:admin:admin,1,,"Get handle to the HDB"
+before,0,0,q,rdb1Handle:hopen`::41802:admin:admin,1,,"Get handle to RDB1"
+before,0,0,q,rdb2Handle:hopen`::41803:admin:admin,1,,"Get handle to RDB2"
+before,0,0,q,wdbHandle:hopen`::41805:admin:admin,1,,"Get handle to WDB"
+before,0,0,q,gwHandle:hopen`::41806:admin:admin,1,,"Get handle to gateway"
```

Comment:
hardcoded ports in unit tests isn't ideal - a future developer should be able to run all unit tests in TorQ easily, without having to faff about with configuring base ports or worrying about port collisions etc.

Reply:
I'm having some trouble resolving this issue. I've tried getting the baseport from the system variable and then joining it with sv, but this gives `::/41800/:admin:admin. I'm not sure where to go from here.

Reply:
So couple of thoughts - 

1. It should be fairly simple to do string manipulation without using `sv` e.g.

```q
`$"::",getenv[`BASEPORT],":admin:admin"
```

2. TorQ has functions for getting connections to other processes based on proctype/procname without manually constructing connection strings etc. - if we use those, we shouldn't have to worry about any of this